superset-frontend/spec/javascripts/dashboard/components/HeaderActionsDropdown_spec.jsx

```diff
@@ -79,19 +79,19 @@ describe('HeaderActionsDropdown', () => {
       expect(menu.find(SaveModal)).not.toExist();
     });
 
-    it('should render five Menu items', () => {
+    it('should render four Menu items', () => {
```

Comment:
I changed to "should render available Menu items" since this is nested in a describe with the role being tested. This way future changes are not going to impact this text.